src/main/java/com/manardenza/console/InputType.java

```diff
@@ -0,0 +1,32 @@
+package com.manardenza.console;
+
+import lombok.Getter;
+import org.apache.commons.lang3.StringUtils;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.function.Predicate;
+
+public enum InputType {
+    STRING(input -> true, ""),
+    INTEGER(input -> StringUtils.isNumeric(input), "Error! Please enter a number"),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1612)